### PR TITLE
Implement/#3 implement 01 add config sample

### LINKED ISSUE #3: Implement 01 Add config sample

### PR DESCRIPTION
**Pull request by Nasdan**
Close #3

### CODE REVIEW

File: frontend/01 Add config/README.md [hunk 1]
```diff
@@ -16,6 +23,364 @@ npm install
...
+- [@types/jest](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/df38f202a0185eadfb6012e47dd91f8975eb6151/types/jest): Typings for jest.
+- [ts-jest](https://github.com/kulshekhar/ts-jest): A preprocessor with sourcemap support to help use Typescript with Jest.
+- [raf](https://github.com/chrisdickinson/raf): `requestAnimationFrame` polyfill for node and browser. Needed for React >= 16
+
+```bash
+npm install jest @types/jest ts-jest raf -D
+
+```
+
+### Render React component libs:
```
Comment (crsanti) [inline frontend/01 Add config/README.md]: React components rendering libs?

File: frontend/01 Add config/README.md [hunk 2]
```diff
@@ -16,6 +23,364 @@ npm install
...
+
+We have two options for render components: `react-test-renderer` or `enzyme`.
```
Comment (crsanti) [inline frontend/01 Add config/README.md]: for rendering / to render

File: frontend/01 Add config/README.md [hunk 3]
```diff
@@ -16,6 +23,364 @@ npm install
...
+  - Pros: no necessary other library to snapshot testing
+  - Cons: hard to test DOM events or other manipulation.
+  - [Sample](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest)
+
+- [enzyme](https://github.com/airbnb/enzyme): Testing utility for React that makes it easier to work with DOM.
+  - Pros: intuitive API for DOM manipulation.
+  - Cons: it needs to install other libraries to snapshot testing.
+
+- [@types/enzyme](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/enzyme): Typings for enzyme.
+- [enzyme-to-json](https://github.com/adriantoine/enzyme-to-json): Convert enzyme wrappers to snapshot testing.
```
Comment (crsanti) [inline frontend/01 Add config/README.md]: Serialize enzyme wrappers for snapshot testing

**Comment by Nasdan** [comment]
Thank you for feedback

**Pull request merged**